File: slot_machine.java
import java.util.Random;
import java.util.Scanner;
public class sloth_machie {
    public static void main(String[] args){
         while(true){
            System.out.println("int the number of spin : ");
            sloth();
         }
    }
    public static void sloth(){
        int a,b,c = 0;
        int win=0;
	int lose=0;
        Random rand = new Random();
        Scanner ingresso = new Scanner(System.in);
	int limite = ingresso.nextInt();
        for(int d=0;d<limite;++d){
		System.out.println("----"+(d+1)+"----");
            for(int e=0;e<=5;++e){
                a=rand.nextInt(4);
                b=rand.nextInt(4);
                c=rand.nextInt(4);
                System.out.println("  "+a+","+b+","+c);
		if(e>=5){
	 		if(a==b && b==c){
				System.out.println("you won");
				win += 1; 
				}
			else{
				System.out.println("you lose");
				lose += 1;
				}
        		}
    		}
	}
	System.out.println("you won for : "+win+" you lose for : "+lose);
    }
}

File: slot_machine.go
package main

import (
	"fmt"
	"math/rand"
)

func scan() {
	var a [5]int
	var b [5]int
	var c [5]int
	var win int
	var lose int
	fmt.Println("\nPut the number of spin\n")
	var ingresso int
	fmt.Scan(&ingresso)
	for i := 0; i < ingresso; i++ {
		fmt.Printf("---%d---\n", i)
		for j := 0; j < 5; j++ {
			a[j] = 1 + rand.Intn(3)
			b[j] = 1 + rand.Intn(3)
			c[j] = 1 + rand.Intn(3)
			fmt.Printf(" %d-%d-%d\n", a[j], b[j], c[j])
		}
		if a[4] == b[4] {
			if b[4] == c[4] {
				win++
			} else {
				lose++
			}
		} else {
			lose++
		}
	}
	fmt.Printf("you won for : %d you lose for : %d", win, lose)
}
func main() {
	for {
		scan()
	}
}
